### DISCUSSION TITLE
ModuleNotFound x509

### DISCUSSION THREAD

**Discussion by Rolig4n** [andreroggeri/pynubank#399] [Q&A]
![image](https://github.com/andreroggeri/pynubank/assets/56307411/6aa94b13-b1fd-4561-a715-ec92ecff284a)
a imagem a cima mostra todos os pacotes que foram instaladas quando executei o pip do pynubank, e ao tentar executar ```pynubank``` para gerar o sertificado, gero a seguinte exception:
![image](https://github.com/andreroggeri/pynubank/assets/56307411/d8ded52d-11db-43ce-be51-ad6247433199)

alguem sabe me dizer o que pode ter acontecido?

**Comment by andreroggeri** [comment]
Me parece que você tem uma versão incompativel da biblioteca `urllib`.

Pelo print você tem instalado a versão `1.24.3` mas ela é incompatível com a biblioteca `requests-pkcs12`.

Poderia tentar instalar uma versão mais nova dela? `pip install urllib3==1.26.15` por ex
